using_images/other_images/jenkins.adoc

```diff
@@ -425,17 +425,109 @@ scripts for your image.
 
 The Jenkins image also provides auto-discovery and auto-configuration
 of slave images for the
-Kubernetes plug-in. The Jenkins image searches for these in the existing image streams within the project that it is running in. The search specifically looks for image streams that have the label `role` set to `jenkins-slave`.
+Kubernetes plug-in. The Jenkins image, via the link:https://github.com/openshift/jenkins-sync-plugin[OpenShift Sync plug-in],
+on Jenkins start-up searches for these in the existing image streams within the project that it is running in, along with any additional
+projects listed in the plug-in's configuration. The search specifically looks for:
```

Comment:
it's actually either it searches the project it's running in, OR it searches the projects listed in the configuration, i believe.  If you want to search your own project and another one, you need to list both.